website/statistics.py

```diff
@@ -340,6 +924,181 @@ def _calc_error_rate(fail, success):
     return (failed * 100) / max(1, failed + successful)
 
 
+def _determine_bool(bool_str):
+    if bool_str == "True":
+        return True
+    return False
```

Comment:
This is rather silly, but a cleaner way to do this is the following:

```suggestion
return bool_str == "True":
```